csharp-advanced-topics/QueueInCSharp/QueueInCSharp/QueueOperations.cs

```diff
@@ -0,0 +1,69 @@
+﻿using QueueInCSharp.Models;
+
+namespace QueueInCSharp;
+
+public static class QueueOperations
+{
+    //declare a queue of orders
+    public static Queue<Order> Orders;
+
+    static QueueOperations()
+    {
+        //initialize the queue
+        Orders = new Queue<Order>();
+
+        //create order objects
+        var Order1 = new Order("Ana", new string[] { "Chockolate", "Coffee" }, 20);
+        var Order2 = new Order("George", new string[] { "Juice", "Sandwich" }, 15);
+        var Order3 = new Order("Bob", new string[] { "Ice cream" }, 5);
```

Comment:
As I've mentioned in the article review, please check all the local variables and correct their names. A lot of variables are named incorrectly.